tests/unit/test_context.py

```diff
@@ -183,6 +183,7 @@ def assert_has_keys(
     'invocation_id',
     'modules',
     'flags',
+    'print'
```

Comment:
Was this addition needed b/c of this PR https://github.com/dbt-labs/dbt-core/pull/4701?

Reply:
If so, I'm guessing the other adapters might need this update as well

Reply:
It was. I'll merge this PR and look at the others

Reply:
Doubled checked snowflake, bigquery, and spark. The unit tests of those are passing just fine! No additional fixes needed for this issue looks like. 😄